coreos-assembler

```diff
@@ -4,6 +4,11 @@ set -euo pipefail
 # Currently this just wraps the two binaries we have today
 # under a global entrypoint with subcommands.
 
+id=$(id -u)
+if test "${id}" = 0; then
+    exec runuser -u builder -- $0 "$@"
+fi
```

Comment:
so this says if I'm root then run as builder user? 

if you aren't root then what happens?

Reply:
Nothing?  If you're running via docker/podman and have already changed user.   If it's something else then you get to keep both pieces.

(Big picture we could probably try to support a non-hardcoded `builder` user, that would help me actually, but I'm not sure I want to mix that into this PR)

Reply:
yeah I guess I was just wondering if we should error out or something instead of continuing